drivers/clock_control/clock_control_nrf.c

```diff
@@ -427,17 +426,64 @@ static void lfclk_spinwait(nrf_clock_lfclk_t t)
 {
 	nrf_clock_domain_t d = NRF_CLOCK_DOMAIN_LFCLK;
 	nrf_clock_lfclk_t type;
+	static const bool lfxo_target =
+		    IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL) |
```

Comment:
These should be `||` since the operands and result are essentially boolean.

Same in other locations (`isr_mode` next definition)

Reply:
done